pkg/policyfilter/state.go

```diff
@@ -235,7 +237,8 @@ func (m *state) updatePodHandler(pod *v1.Pod) {
 	}
 
 	namespace := pod.Namespace
-	err = m.UpdatePod(PodID(podID), namespace, containerIDs)
+	podLabels := labels.FromPod(pod)
```

Comment:
nit: Do we need function `labels.FromPod`? Otherwise, it would be `podLabels := pod.Labels` which is similar to what we do for example with Namespace in the previous line.